students/HarveyHerela/lesson07/html_render.py

```diff
@@ -0,0 +1,128 @@
+class Element:
+    """Base class for all HTML tags."""
+    def __init__(self, tag_name='element', content=None, **kwargs):
+        if content is None:
```

Comment:
I would make `tag_name` a class attribute, rather than instance attribute since it should be the same value for all instances of your `Element` class